src/ci/bin/testCentaurEngineUpgradeLocal.sh

```diff
@@ -6,11 +6,15 @@ export CROMWELL_BUILD_OPTIONAL_SECURE=true
 # shellcheck source=/dev/null
 source "${BASH_SOURCE%/*}/test.inc.sh" || source test.inc.sh
 
-cromwell::build::setup_common_environment
+if [ "${CROMWELL_BUILD_PROVIDER}" = "${CROMWELL_BUILD_PROVIDER_TRAVIS}" -a  -n "${TRAVIS_PULL_REQUEST_BRANCH}" ]; then
```

Comment:
This might be the right way to do things, but it seems odd to me to see `=` in the same expression as `-a` and `-n`?

Reply:
Hopefully more palatable now as a result of your earlier Shellcheck suggestion.